test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -12,7 +12,7 @@ def setup
 
     @options = {
       :order_id => '1',
-      :billing_address => address(:country_name => 'United States of America'),
+      :billing_address => address(:country_name => 'Canada'),
```

Comment:
This was giving a "incompatible country data" error so it needed to be in line with the default test card data.